src/components/TowerDetails.js

```diff
@@ -30,14 +69,56 @@ export const TowerDetails = observer(function TowerDetails(props) {
     const inviteHandler = () => {
         // Get an invite code from the tower
         setInviteButtonState(INVITE_WAITING);
-        generateInviteCode(props.tower.id).then(invite => {
-            navigator.clipboard.writeText(invite.inviteCode);
-            setInviteButtonState(INVITE_COPIED);
-        });
+        setIsInviteCustomizerOpen(true);
+    };
+
+    // Called when we finish configuring the invite
+    const generateInvite = () => {
+        const {time,units} = AVAILABLE_EXPIRY_TIMES[parseInt(inputInviteDuration.current.value) || 0];
+        const selectedUses = parseInt(inputInviteDuration.current.value) || 0;
```

Comment:
```suggestion
        const selectedUses = parseInt(inputInviteUses.current.value) || 0;
```
Fixes a typo that caused the index of the selected expiry option to be sent as the number of invite uses.